src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -1555,27 +1555,6 @@ describe("Decimal", () => {
           });
         }
       );
-
```

Comment:
We should be able to re-introduce this test now that you've re-added the invariant.

Reply:
Cheers for spotting this. I thought I added all the `invariant` tests back in to the specs file. Must have missed this. My bad.